content/docs/intro/console/accounts-and-organizations/organizations.md

```diff
@@ -67,7 +67,7 @@ relref "/docs/guides/saml" >}}).
 
 ### GitHub-backed
 
-To add a GitHub-backed organization to Pulumi, an administrator of the GitHub organization
+To add a GitHub-backed organization to Pulumi, an admin of the GitHub organization
```

Comment:
Little thing, but since this PR is about changing Console terminology and these words instead refer to GitHub, GitLab and Atlassian "admin-type things", we should probably make sure we're using the right terminology for each provider, just to avoid any confusion.

Reply:
on very quick google search they all call them admins